backend/x86_gas.ml

```diff
@@ -105,7 +105,8 @@ let i0 b s = bprintf b "\t%s" s
 let i1 b s x = bprintf b "\t%s\t%a" s arg x
 let i1_s b s x = bprintf b "\t%s%s\t%a" s (suf x) arg x
 let i2 b s x y = bprintf b "\t%s\t%a, %a" s arg x arg y
-let i2_s b s x y = bprintf b "\t%s%s\t%a, %a" s (suf y) arg x arg y
+let i2_sx b s x y = bprintf b "\t%s%s\t%a, %a" s (suf x) arg x arg y
```

Comment:
I think you should keep pre-existing functions with the same (bad) name to avoid unrelated changes in this PR.

Reply:
:+1: